introduction/report-structure.md

```diff
@@ -0,0 +1,18 @@
+# Report structure
+To anwser the four main subquestions, the report will have four sections.  
+The first subquestion zooms in on what can be learned from (non-software) communities.
+Before theories can be defined about what can be learned from all sorts of communities, it must be defined what a community is.
+Because communities exist in different sizes, with different structures and on different platforms, a few of these communities will be analysed in section 1.
+
+Software development in its current form has some methodologies on how requirements are established for a (big) software development project.
+*** referentie naar subvraag ***  
+These requirements are the basis of how the software program will be build and are therefore an essential part of the software development process. In the context of this project, the requirements are build from the problem-description and thus must be correctly translated.
+In section 2, a few of the method wich are used to establish requirements will be discribed.
+The dynamics of the requirements, like management and implementation, will also be given a closer look.
+
+The platform that will be build based on this research will probably not be the only attempt to involve a software community in building software.
+The third subquestion is therefore focussed on existing platforms where software communities are on way or another involved in the software development process.
+Section 3 will focus on the structure of the existing software communities. The techniques used by those communities and the way that the development process engages the community will also be examined.
+
+The first three subquestion focus on what should be done to create a platform where the community is involved in the development process.
+However, because the platform that will be developed is an extention of the current FeedbackFruits-platform, the FeedbackFruits-platform may pose some limitations. With subquestion four, the structure of the FeedbackFruits-platform will be examined, as well as the extensiblity of the platform. The result of this subquestion can be found in section 4.
```

Comment:
Again, no need to mention a platform. The solution the problem however or whatever it is, should be an extension of the current platform indeed